build/repo.targets

```diff
@@ -246,14 +246,8 @@
   </Target>
 
   <Target Name="_RemoveTimestampFromDepsFile">
-    <GetOSPlatform>
```

Comment:
It's missing because `dotnet msbuild KoreBuild.proj` and `dotnet msbuild repo.targets` are not the same. Building repo.targets directly doesn't import the build tasks assembly.

FYI - MSBuild 15.3 has a new [helper method](https://github.com/Microsoft/msbuild/blob/v15.3.407.29267/src/Build/Evaluation/IntrinsicFunctions.cs#L431-L439).
`Condition=" $([MSBuild]::IsOSPlatform('OSX')) OR $([MSBuild]::IsOSPlatform('Windows')) OR $([MSBuild]::IsOSPlatform('Linux'))"`